cloud_governance/policy/ec2_stop.py

```diff
@@ -99,13 +99,13 @@ def __trigger_mail(self, tags: list, stopped_time: str, resource_id: str, days:
         receivers_list.append(account_admin)
         cc.append(account_admin)
         subject = f'cloud-governance alert: delete ec2-stop more than {days} days'
-        content = 'This instance will be deleted after 30 days'
+        content = 'This instance will be deleted after 30 days, if cloud-governance won\'t delete it please add Policy=Not_Delete to your tags. If you already added ignore this.'
```

Comment:
please add Policy=Not_Delete to your tags, for skipping this policy. If you already added ignore this.